server/src/routes/logout.c

```diff
@@ -0,0 +1,34 @@
+/*
+** EPITECH PROJECT, 2024
+** MyTeams
+** File description:
+** No file there , just an epitech header example .
+** You can even have multiple lines if you want !
+*/
+
+#include "server.h"
+#include "json/json.h"
+#include "server_utils.h"
+
+response_t logout_route(request_t *request, void *data)
+{
+    roundtable_server_t *server = (roundtable_server_t *)data;
+    json_object_t *json = (json_object_t *) json_parse(request->body);
+    char *user_uuid = NULL;
+    roundtable_client_t *client = NULL;
+
+    if (strcmp(request->route.method, "POST") != 0)
```

Comment:
You didn't check the method in login.c. It would be better to do it

Reply:
I don't have enough line but I will try